binding-mruby/font-binding.cpp

```diff
@@ -135,17 +142,21 @@ DEF_KLASS_PROP(Font, mrb_bool, DefaultItalic,  "b", bool)
 DEF_KLASS_PROP(Font, mrb_bool, DefaultOutline, "b", bool)
 DEF_KLASS_PROP(Font, mrb_bool, DefaultShadow,  "b", bool)
 
-MRB_FUNCTION(FontGetDefaultName)
+MRB_METHOD(FontGetDefaultName)
```

Comment:
It's what you said; only a method has an object to operate on (self), functions don't. The `FUNCTION` macro was to avoid having to `(void)` the self argument every time in module functions.